packages/fuel-gauge/src/doc-examples.test.ts

```diff
@@ -6,19 +6,19 @@ import {
   bn,
   Provider,
   hashMessage,
-  NativeAssetId,
   Address,
   arrayify,
   hexlify,
   randomBytes,
   getRandomB256,
-  ZeroBytes32,
   addressify,
   Contract,
   Wallet,
   WalletUnlocked,
   Signer,
   ContractFactory,
+  ZeroBytes32,
+  NativeAssetId,
```

Comment:
this ordering changes in imports are related to specific config inside your vscode, right @Dhaiwat10?

project should contain a vscode config file, and everyone should use the same... if this is a good config, should go to the project vscode config file.

otherwise next person that change this file can change the order again, and we end up reviewing ordering changes in a lot of prs... same follows for all lints/formatting configs

Reply:
this can be done on other small PR only to update vscode config file... just pointing it out as different configs between the team generates unnecessary reviews

Reply:
@LuizAsFight good catch. I don't think it's because of my vscode settings but rather because during this PR, I changed where these constants get imported from multiple times and then we eventually ended up resetting to our initial API. 

I don't think we have a setting in our vscode settings file in the repo to automatically arrange imports in a certain way, though. That would've taken care of this.

Want me to create a new issue for this?